src/Posts/Menu.jsx

```diff
@@ -77,6 +77,44 @@ const moderate = (account, block, messageKey, action) => {
   });
 };
 
+const promoteToBlog = () => {
+  if (state.loading) {
+    return;
+  }
+
+  if (!context.accountId && props.requestAuthentication) {
+    props.requestAuthentication();
+    return;
+  } else if (!context.accountId) {
+    return;
+  }
+
+  State.update({
+    loading: true,
+  });
+
+  const data = {
+    index: {
+      promote: JSON.stringify({
+        key: context.accountId,
+        value: {
+          operation: "addToBlog",
```

Comment:
```suggestion
          type: "blog",
```
After thinking about it more, maybe this could be type: "blog" the operations are: add, delete. add can be the default.